test/BidiCharacterTest.txt

```diff
@@ -80,12 +80,12 @@
 202E 202D 05D0 202C 0028 005B 202C 202B 0061 005D 0029 0062;2;1;x x 4 x 3 3 x x 4 4 4 4;8 9 10 11 5 4 2
 
 # Nonspacing marks applied to paired brackets
-0061 0028 0062 0029 0331;1;1;2 2 2 2 2;0 1 2 3 4
-0061 0028 0332 0062 0029 0333;1;1;2 2 2 2 2 2;0 1 2 3 4 5
-05D0 0028 05D1 0029 0331;0;0;1 1 1 1 1;4 3 2 1 0
-05D0 0028 0332 05D1 0029 0333;0;0;1 1 1 1 1 1;5 4 3 2 1 0
-0661 0028 0662 0029 0331;0;0;2 1 2 1 1;4 3 2 1 0
-0661 0028 0332 0662 0029 0333;0;0;2 1 1 2 1 1;5 4 3 2 1 0
+0061 0028 0062 0029 0331;1;1;2 2 2 1;3 0 1 2
```

Comment:
I believe you are not supposed to touch this file it is imported directly from the Unicode bidi spec

Reply:
Do you think BidiCharacterTest.js should also use `string.split('')`